.dockerignore

```diff
@@ -2,5 +2,4 @@ Dockerfile
 docker-compose.yml
 tsconfig.tsbuildinfo
 node_modules
-/built/
```

Comment:
I think this should be reverted as we shouldn't need to pass `/built/` into the Docker build context